mesonbuild/compilers/vala.py

```diff
@@ -54,6 +55,11 @@ def get_no_warn_args(self):
     def get_werror_args(self):
         return ['--fatal-warnings']
 
+    def get_colorout_args(self, colortype):
+        if version_compare(self.version, '>=0.37.0'):
```

Comment:
The commit is in 0.37.1 or later: https://github.com/GNOME/vala/commit/e1b399b332dbf0875b6de8499f31877db6f9f3a